bosh/opsfiles/rules.yml

```diff
@@ -356,4 +356,27 @@
         severity: critical
       annotations:
         summary: "CF Application `{{$labels.environment}}/{{$labels.deployment}}/{{$labels.organization_name}}/{{$labels.space_name}}/{{$labels.application_name}}` does not have any instance running"
-        description: "CF Application `{{$labels.application_name}}` at environment `{{$labels.environment}}`, deployment `{{$labels.deployment}}`, org `{{$labels.organization_name}}`, and space `{{$labels.space_name}}` has not had any instance running. You might want to look into that."
\ No newline at end of file
+        description: "CF Application `{{$labels.application_name}}` at environment `{{$labels.environment}}`, deployment `{{$labels.deployment}}`, org `{{$labels.organization_name}}`, and space `{{$labels.space_name}}` has not had any instance running. You might want to look into that."
+
+# Network watchers
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/name=prometheus2/properties/prometheus/custom_rules?/-
+  value:
+    name: networker
+    rules:
+    - alert: NetworkTrafficBurst
+      expr: delta(firehose_counter_event_gorouter_total_requests_total{bosh_job_name="router"}[5m]) >= 10000
```

Comment:
This might be more futureproof as a percentage increase

Reply:
Yeah, we can definitely do that - I just didn't think about it off the top of my head, we prolly should though.